sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowingStrategies.java

```diff
@@ -122,8 +126,13 @@ public static ClosingBehavior fromProto(RunnerApi.ClosingBehavior proto) {
   }
 
   // This URN says that the coder is just a UDF blob the indicated SDK understands
+  // TODO: standardize such things
   private static final String CUSTOM_CODER_URN = "urn:beam:coders:custom:1.0";
 
+  // This URN says that the WindowFn is just a UDF blob the indicated SDK understands
+  // TODO: standardize such things
+  private static final String CUSTOM_WINDOWFN_URN = "urn:beam:windowfn:custom:1.0";
+
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
   public static RunnerApi.MessageWithComponents toProto(WindowFn<?, ?> windowFn)
```

Comment:
The only coder in the `Components` will be the `WindowCoder`, as implemented - is it worth specifying that?